src/object_store/src/object/s3.rs

```diff
@@ -293,7 +294,9 @@ impl StreamingUploader for S3StreamingUploader {
 
 fn get_upload_body(data: Vec<Bytes>) -> ByteStream {
     SdkBody::retryable(move || {
-        Body::wrap_stream(stream::iter(data.clone().into_iter().map(ObjectResult::Ok))).into()
+        SdkBody::from_body_0_4(Body::wrap_stream(stream::iter(
```

Comment:
where does `from_body_0_4` come from? Any documentation?

Reply:
Fixed. Now it uses `hyper-0-14-x`, which matches our deps.